portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -24,13 +24,16 @@
   private final String email;
   private final long timestampMillis;
   private final String commentInput;
+  private final String fileUrl;
```

Comment:
if this is indeed an Optional value, as you have the `Optional` values in `DataServlet`, consider making it also Optional in the Comment class? this differentiates between strings that are 'empty' vs things that are 'optional' and does not need to exist.


Not sure if datastore entity allows for `Optional`

Reply:
Datastore doesn't support Optional sadly... I add a placeholder image in script.js if a Comment doesn't have an attached image

Reply:
i see, sg.